openedx/adg/lms/applications/helpers.py

```diff
@@ -103,3 +105,104 @@ def check_validations_for_current_record(attrs, error_message):
         errors['date_completed_year'] = _(error_message).format(key='Date completed year')
 
     return errors
+
+
+def is_displayable_on_browser(file):
+    """
+    Check if the input file can be displayed as an embedded view on a browser
+
+    Arguments:
+        file (FieldFile): file to be checked
+
+    Returns:
+        bool: False if file type is 'doc', True otherwise
+    """
+    filename = str(file).lower()
+    if filename.endswith('.doc'):
+        return False
+
+    return True
+
+
+def get_embedded_view_html(file):
+    """
+    Return html to display file in browser
+
+    Arguments:
+        file (File): file that needs to be rendered
+
+    Returns:
+        SafeText: HTML to display image or embedded view for pdf
+    """
+    _, ext = os.path.splitext(str(file))
+    if ext in ['.jpg', '.png']:
+        html = '<img src="{path_to_image}"/>'.format(path_to_image=file.url)
+    else:  # for pdf
+        html = '<iframe src="{path_to_file}" style="width:600px; height:500px;"></iframe>'.format(path_to_file=file.url)
+
+    return format_html(html)
+
+
+def get_duration(entry, is_current):
+    """
+    Extract, format and return start and end date of Education/WorkExperience
+
+    Arguments:
+        entry (UserStartAndEndDates): Education or WorkExperience object
+        is_current (bool): True if Education is in progress or WorkExperience is current position
+
+    Returns:
+        str: start and end date
+    """
+    start_date = '{month} {year}'.format(month=entry.get_date_started_month_display(), year=entry.date_started_year)
+    completed_date = _('Present') if is_current else '{month} {year}'.format(
+        month=entry.get_date_completed_month_display(), year=entry.date_completed_year
+    )
+    return '{started} {to} {completed}'.format(started=start_date, to=_('to'), completed=completed_date)
+
+
+def _get_application_review_info(application):
+    """
+    Get application review information if the application has been reviewed, i.e. application status is not 'open'
+
+    Arguments:
+        application (UserApplication): User application
+
+    Returns:
+        reviewed_by (str): Name of reviewer
+        review_date (str): Date of review submission
+    """
+    reviewed_by = None
+    review_date = None
+    if application.status != application.OPEN:
+        reviewed_by = application.reviewed_by.get_full_name()
```

Comment:
We don't use first name and last name or take it from user at any stage of registration. Can we use the full_name from user profile? We store full name in profile at registration. First and last name are empty.

Reply:
Good point. But in this case we are talking about the admin user. The user in `application.reviewed_by` is an ADG or more specifically a BU admin. And we're not registering admins in our application, as of yet, right?

Reply:
But most likely for admins we will also be using user profile name. So in any case, I have updated it according to your suggestion.